main.go

```diff
@@ -56,6 +56,135 @@ var (
+	frameworkResourcesUsedCpus = prometheus.NewDesc(
+		"mesos_framework_resources_used_cpus",
+		"Fractional CPUs used by a framework",
+		frameworkLabels, nil,
+	)
+
+	frameworkResourcesUsedDisk = prometheus.NewDesc(
+		"mesos_framework_resources_used_disk",
+		"Disk space used by a framework",
+		frameworkLabels, nil,
+	)
+
+	frameworkResourcesUsedMemory = prometheus.NewDesc(
+		"mesos_framework_resources_used_memory",
+		"Memory used by a framework",
+		frameworkLabels, nil,
+	)
+
+	masterMetricsLabels = []string{"host"}
+
+	masterMetrics = []snapshotMetric{
+		snapshotMetric{
+			desc: prometheus.NewDesc(
+				"mesos_master_tasks_error_total",
+				"Number of tasks that have errored",
+				masterMetricsLabels, nil,
+			),
+			snapshotKey: "master/tasks_error",
+			valueType:   prometheus.CounterValue,
+		},
+		snapshotMetric{
+			desc: prometheus.NewDesc(
+				"mesos_master_tasks_failed_total",
+				"Number of tasks that have failed",
+				masterMetricsLabels, nil,
+			),
+			snapshotKey: "master/tasks_failed",
+			valueType:   prometheus.CounterValue,
+		},
+		snapshotMetric{
+			desc: prometheus.NewDesc(
+				"mesos_master_tasks_finished_total",
+				"Number of tasks that have finished",
+				masterMetricsLabels, nil,
+			),
+			snapshotKey: "master/tasks_finished",
+			valueType:   prometheus.CounterValue,
+		},
+		snapshotMetric{
+			desc: prometheus.NewDesc(
+				"mesos_master_tasks_killed_total",
+				"Number of tasks that got killed",
+				masterMetricsLabels, nil,
+			),
+			snapshotKey: "master/tasks_killed",
+			valueType:   prometheus.CounterValue,
+		},
+		snapshotMetric{
+			desc: prometheus.NewDesc(
+				"mesos_master_tasks_lost_total",
+				"Number of tasks that got lost",
+				masterMetricsLabels, nil,
+			),
+			snapshotKey: "master/tasks_lost",
+			valueType:   prometheus.CounterValue,
+		},
+		snapshotMetric{
+			desc: prometheus.NewDesc(
+				"mesos_master_tasks_running",
+				"Number of tasks that are running",
+				masterMetricsLabels, nil,
+			),
+			snapshotKey: "master/tasks_running",
+			valueType:   prometheus.GaugeValue,
+		},
+		snapshotMetric{
+			desc: prometheus.NewDesc(
+				"mesos_master_tasks_staging",
+				"Number of tasks that are staging",
+				masterMetricsLabels, nil,
+			),
+			snapshotKey: "master/tasks_staging",
+			valueType:   prometheus.GaugeValue,
+		},
+		snapshotMetric{
+			desc: prometheus.NewDesc(
+				"mesos_master_tasks_starting",
+				"Number of tasks that are starting",
+				masterMetricsLabels, nil,
+			),
+			snapshotKey: "master/tasks_starting",
+			valueType:   prometheus.GaugeValue,
+		},
+	}
+
+	slaveMetricsLabels = []string{"host"}
+
+	slaveMetrics = []snapshotMetric{
+		snapshotMetric{
+			desc: prometheus.NewDesc(
+				"mesos_slave_cpus_total",
+				"CPUs advertised by a Mesos Slave",
+				slaveMetricsLabels, nil,
+			),
+			snapshotKey: "slave/cpus_total",
+			valueType:   prometheus.GaugeValue,
+		},
+		snapshotMetric{
+			desc: prometheus.NewDesc(
+				"mesos_slave_disk_total",
```

Comment:
Same here, and also add a unit suffix: `mesos_slave_disk_bytes` or `mesos_slave_total_disk_bytes` (if that's clearer, as long as there's no `_total` as a suffix).